src/app/SharpRaven/RavenClient.cs

```diff
@@ -363,6 +393,19 @@ protected virtual string Send(JsonPacket packet)
             }
             catch (Exception exception)
             {
+                if (!cachePacketsWhenOffline) return HandleException(exception, requester);
+                try
+                {
+                    if (!string.IsNullOrEmpty(packet.Message)) packet.Message += "\n";
+                    packet.Message +=
+                        $"Attempted to send on {DateTime.UtcNow.ToShortDateString()} {DateTime.UtcNow.ToLongTimeString()} UTC however packet failed to send and was cached locally.";
```

Comment:
Can you explain what "Extra" is ? Is it another field accompanied with the message? If so then yes that would definitely be more appropriate.